packages/jsdoc-plugins/lib/relation-fixer/addmissingdoclets.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * @license Copyright (c) 2017, CKSource - Frederico Knabben. All rights reserved.
+ * Licensed under the terms of the MIT License (see LICENSE.md).
+ */
+
+'use strict';
+
+const isNonEmptyArray = require( './utils/isnonemptyarray' );
```

Comment:
I have mixed feelings regarding implementing special modules for such basic functions. Fixing duplicated code is easier than fixing broken abstraction. It's simply cheaper at the beginning to have `isNonEmptyArray` defined in two places where it's used and it's cheaper later on when it turns out that after some changes it's used in one or zero places :P.